llvm/test/CodeGen/AIE/aie2/end-to-end/TahnTemplated-swp.ll

```diff
@@ -0,0 +1,431 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+;
+; This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+; See https://llvm.org/LICENSE.txt for license information.
+; SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+;
+; (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
+
+; RUN: llc -O2 -mtriple=aie2 %s -o - | FileCheck %s --check-prefix=WAW-STICKY-ON
+; RUN: llc -O2 -mtriple=aie2 -aie-pipeliner-waw-sticky-registers=false \
+; RUN:        %s -o - | FileCheck %s  --check-prefix=WAW-STICKY-OFF
+; RUN: llc -O2 -mtriple=aie2 -debug-only=pipeliner %s -o - 2>&1 > /dev/null \
+; RUN:       | FileCheck %s --check-prefix=WAW-STICKY-ON-PIPELINER
```

Comment:
Nit: I'd rather have a single set of check lines (with the new combiner on), otherwise it makes the test really long

Reply:
I reduced the test.